btr-api/src/btr_api/config.py

```diff
@@ -104,6 +104,7 @@ class Config:  # pylint: disable=too-few-public-methods
     JWT_OIDC_FIRSTNAME = os.getenv('JWT_OIDC_FIRSTNAME', 'firstname')
     JWT_OIDC_LASTNAME = os.getenv('JWT_OIDC_LASTNAME', 'lastname')
 
+    AUTH_SVC_URL = os.getenv('AUTH_SVC_URL', '')
```

Comment:
hey so this should follow the same pattern as below so that we're using the same shared variables in our deployment as all the other projects (sorry I thought this was already getting set in the config):
`os.getenv('AUTH_API_URL', '') + os.getenv('AUTH_API_VERSION', '')`